docs/resources/aws_ssm_activation.md

```diff
@@ -0,0 +1,76 @@
+---
+title: About the aws_ssm_activation Resource
+platform: aws
+---
+
+# aws\_ssm\_activation
+
+Use the `aws_ssm_activation` InSpec audit resource to test properties of a ssm activation.
+
+## Syntax
+
+ An `aws_ssm_activation` resource block uses the parameter to select a ssm activation.
+
+    describe aws_ssm_activation(name: 'ssm-activation-name-1234') do
```

Comment:
Should the parameter name be `activation_id` throughout the document?

Reply:
Updated in recent commit.